firefox-ios/Client/Frontend/Browser/TermsOfUse/TermsOfUseViewController.swift

```diff
@@ -0,0 +1,262 @@
+                sheetContainer.trailingAnchor.constraint(lessThanOrEqualTo: view.trailingAnchor, constant:
+                                                            -UX.sheetContainerSidePadding)
+            ])
+        } else {
+            constraints.append(contentsOf: [
+                sheetContainer.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+                sheetContainer.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+            ])
+        }
+
+        NSLayoutConstraint.activate(constraints)
+    }
+    private func setupGrabber() {
+        grabberView.backgroundColor = UIColor.systemGray3
+        grabberView.layer.cornerRadius = UX.grabberHeight / 2
+        grabberView.translatesAutoresizingMaskIntoConstraints = false
+        sheetContainer.addSubview(grabberView)
+
+        NSLayoutConstraint.activate([
+            grabberView.topAnchor.constraint(equalTo: sheetContainer.topAnchor, constant: UX.grabberTopPadding),
+            grabberView.centerXAnchor.constraint(equalTo: sheetContainer.centerXAnchor),
+            grabberView.widthAnchor.constraint(equalToConstant: UX.grabberWidth),
+            grabberView.heightAnchor.constraint(equalToConstant: UX.grabberHeight)
+        ])
+    }
+
+    private func setupContentStack() {
+        stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = UX.stackSpacing
+        stackView.alignment = .fill
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        sheetContainer.addSubview(stackView)
+
+        NSLayoutConstraint.activate([
+            stackView.leadingAnchor.constraint(equalTo: sheetContainer.leadingAnchor, constant: UX.stackSidePadding),
+            stackView.trailingAnchor.constraint(equalTo: sheetContainer.trailingAnchor, constant: -UX.stackSidePadding),
+            stackView.topAnchor.constraint(equalTo: grabberView.bottomAnchor, constant: UX.stackSpacing),
+            stackView.bottomAnchor.constraint(equalTo: sheetContainer.bottomAnchor, constant: -UX.stackSidePadding)
+        ])
+
+        [createLogo(), createTitle(), createDescription(), createAcceptButton(), createRemindMeLaterButton()].forEach {
+            stackView.addArrangedSubview($0)
+        }
+    }
+
+    private func createLogo() -> UIImageView {
+        let imageView = UIImageView(image: UIImage(imageLiteralResourceName: ImageIdentifiers.homeHeaderLogoBall))
+        imageView.contentMode = .scaleAspectFit
+        imageView.heightAnchor.constraint(equalToConstant: UX.logoSize).isActive = true
+        imageView.widthAnchor.constraint(equalToConstant: UX.logoSize).isActive = true
+        return imageView
+    }
+
+    private func createTitle() -> UILabel {
+        let label = UILabel()
+        label.text = viewModel.titleText
+        label.font = FXFontStyles.Regular.headline.scaledFont()
+        label.textAlignment = .center
+        label.textColor = currentTheme().colors.textPrimary
```

Comment:
Check if on theme change the color is updated. We usually update the colors in the applyTheme func

Reply:
I moved it to applyTheme func. Also tested on dark mode